src/runtime/runtime.js

```diff
@@ -329,6 +332,14 @@ export class Runtime {
         });
   }
 
+  /** @override */
+  showLoginPrompt() {
+    return this.configured_(true)
+        .then(runtime => {
+          runtime.showLoginPrompt();
```

Comment:
You lost `return` here, looks like. And a unit test should have caught this.

Reply:
done